modules/grub.sh

```diff
@@ -173,7 +173,7 @@ function grub_install_mender_grub() {
     sudo rm -rf work/boot/EFI/systemd
     sudo rm -rf work/boot/NvVars
     for empty_dir in $(
-                     cd work/boot && find . -maxdepth 1 -type d -empty
+                     cd work/boot && find . -maxdepth 1 -type d -empty -not -name .
```

Comment:
Q:
! empty ?

Reply:
Not sure what the question is... You mean use `\! -empty` operator? That's not correct, because we do want empty directories, just not `.` if there's nothing at all.

Reply:
Discussed locally